src/molecule.py

```diff
@@ -500,7 +467,7 @@ def x(self):
             coors = nx.get_node_attributes(self,'x')
             return np.array([coors[i] for i in self.L()])
 except ImportError:
-    logger.warning("Cannot import optional NetworkX module, topology tools won't work\n.")
```

Comment:
The lines highlighted in red are newer, so the changes here need to be reverted, perhaps by merging the master branch into this branch.